Kapacitor: clarify behavior of StateCountNode and StateCountDuration (#2572)
influxdata/docs-v2: content/kapacitor/v1.5/nodes/state_count_node.md

---

The `stateCount` node computes the number of consecutive points in a given state.
The state is defined via a lambda expression.

> For each consecutive point for which the expression evaluates as `true`, the state count will be incremented.

Reviewer: ```suggestion
For each consecutive point the lambda expression evaluates as `true`, the state count is incremented.
```
